contracts/test/TestPaymasterAcceptAll.sol

```diff
@@ -22,8 +24,12 @@ contract TestPaymasterAcceptAll is BasePaymaster {
     internal virtual override view
     returns (bytes memory context, uint256 validationData) {
         (userOp, userOpHash, maxCost);
-        // return a context, as it is used for EntryPoint gas checking.
-        return ("1", 0);
+        if ( callPostOp) {
```

Comment:
Wouldn't it be better to have a different paymaster where you inherit this paymaster and add on a postOp function, instead of carrying a flag? I know we did similar things elsewhere, but it seems more readable to just have a different one.